Yakkity-Yulelogs/reactive-nutshell
# Rc edit own events

# Description

A user can now edit their own events.

Fixes # 20

## Type of change

Please delete options that are not relevant.


# Testing Instructions

git fetch --all
git checkout rc-edit-own-events
confirm that you can edit your own events.
confirm that there is no edit button on other's events.



# Checklist:
